src/vnsw/agent/test/windows/test_util_tap.cc

```diff
@@ -0,0 +1,22 @@
+void DeleteTapByName(const char* if_name) {
```

Comment:
A comment explaining why it's empty could be useful. Maybe `\\ TODO(WINDOWS)`?

Reply:
I'll stick to traditional `// TODO(WINDOWS)`.